src/test/HostActivationTests/GivenThatICareAboutPortableAppActivation.cs

```diff
@@ -328,6 +328,32 @@ public void Framework_Dependent_AppHost_Succeeds()
                 .HaveStdOutContaining($"Framework Version:{sharedTestState.RepoDirectories.MicrosoftNETCoreAppVersion}");
         }
 
+        [Fact]
+        public void Framework_Dependent_DLL_LongPath_Succeeds()
```

Comment:
Better place might be the `GivenThatICareAboutWindowsOsShims` since this is also Windows specific and similar to the version shims which are also handled in the manifest. Maybe we can rename that calls to just `GivenThatICareAboutWindowsSpecificBehavior` or something like that.

Reply:
Sounds good - will move / rename.
As a side note, is there a specific reason that all the tests are GivenThatICareAbout* ? I find it makes everything really hard to read (and that naming scheme doesn't seem to be a well-known pattern / practice that I could find)

Reply:
I'm not aware of any good reason … I knew people in the past who think that test classes + methods should read as English sentences of statements (which the test validates). This seems to follow that logic.

I'm not a fan either, just didn't get around to actually attack this. But let's do that. If you want please rename this windows specific to something much shorter :-)

Reply:
Yay - I'll plan to remove the prefix from all the host activation tests (in a separate change)